includes/classes/InternalConnections/NetworkSiteConnection.php

```diff
@@ -1106,4 +1107,29 @@ public function update_content_via_rest( $new_post_id ) {
 			);
 		}
 	}
+
+	/**
+	 * Exclude additional meta data for network distributions
+	 *
+	 * @hook dt_exclude_additional_meta_data
+	 *
+	 * In network connections the featured image is set prior to the meta data.
+	 * Excluding the `_thumbnail_id` meta from distribution prevents the meta
+	 * data from referring to the attachment ID of the original site.
+	 *
+	 * You can use this filter to exclude additional meta data from the distribution.
+	 *
+	 * @since 2.0.2
```

Comment:
```suggestion
	 * In network connections the featured image is set prior to the meta data.
	 * Excluding the `_thumbnail_id` meta from distribution prevents the meta
	 * data from referring to the attachment ID of the original site.
	 *
	 * @since 2.0.0
```